.travis.yml

```diff
@@ -1,31 +1,40 @@
-language: node_js
-node_js:
-- stable
-branches:
-  only:
-  - master
```

Comment:
Seem to recall that saucelabs doesn't allow testing of PRs

Reply:
Not sure what you mean by this. Anything I should change/fix?

Reply:
For external PRs, the tests will fail (I think). See https://docs.travis-ci.com/user/sauce-connect/:

> Note that due to security restrictions, the Sauce Labs addon is not available on pull request builds

I reckon that's why `.travis.yml` had this restriction to only build master.

We could just keep it as-is and see what happens :)

Reply:
Gotcha. I removed `only: master` because I was testing on other branches. I think @rvagg put it there to begin with just to reduce spam, but I also removed mail notifications so shouldn't spam at all.

Also, we're not using sauce labs addon. It used to be in `.travis.yml` but I removed it since I didn't know what it was used for and turns out we don't need it. Maybe this has to do with using `localtunnel` (which is the one that fails sometimes). Maybe you need this addon to use `--sauce-connect` with `zuul`.

```
lms@x240 ~/src/level/memdown (master)
$ ./node_modules/.bin/zuul -h

  Usage: zuul [options] <files | dir>

  Options:

    -h, --help                             output usage information
    -V, --version                          output the version number
    --ui <testing ui>                      ui for tests (mocha-bdd, mocha-tdd, qunit, tape)
    --local [port]                         port for manual testing in a local browser
    --tunnel [type]                        establish a tunnel for outside access. only used when --local is specified
    --disable-tunnel                       don't establish a tunnel for outside access. override any config in .zuul.yml and .zuulrc
    --phantom [port]                       run tests in phantomjs. PhantomJS must be installed separately.
    --phantom-remote-debugger-port [port]  connect phantom to remote debugger
    --phantom-remote-debugger-autorun      run tests automatically when --phantom-remote-debugger-port is specified
    --electron                             run tests in electron. electron must be installed separately.
    --tunnel-host <host url>               specify a localtunnel server to use for forwarding
    --sauce-connect [tunnel-identifier]    use saucelabs with sauce connect instead of localtunnel. Optionally specify the tunnel-identifier
    --server <the server script>           specify a server script to be run
    --list-available-browsers              list available browsers and versions
    --browser-name <browser name>          specficy the browser name to test an individual browser
    --browser-version <browser version>    specficy the browser version to test an individual browser
    --browser-platform <browser platform>  specficy the browser platform to test an individual browser
    --browser-retries <retries>            number of retries allowed when trying to start a cloud browser, default to 6
    --browser-output-timeout <timeout>     how much time to wait between two test results, default to -1 (no timeout)
    --concurrency <n>                      specify the number of concurrent browsers to test
    --no-coverage                          disable code coverage analysis with istanbul
    --open                                 open a browser automatically. only used when --local is specified
```

Maybe we should explore using `--sauce-connect` instead of `localtunnel` to see if we can get rid of the random test failures

Reply:
Aha, thanks for clarifying.